pkg/commands/testdata/Should scaffold application declaration based on opts.golden

```diff
@@ -26,16 +26,19 @@ image:
 subDomain: my-app
 
 # The port your app listens on
-# Comment this out to avoid setting up a service (required if url is specified)
+# Comment this out to avoid setting up a service (required if subDomain is specified)
 #
 port: 3000
 
 # Enable prometheus scraping of metrics
 #prometheus:
 #  path: /metrics
 
+# Enable connection with a PostgreSQL database
```

Comment:
Should we mention where this name comes from (cluster-declaration)?

Or a bit more advanced, we could insert the cluster declaration's postgres name automatically.

edit: OK, I see the userdoc covers this, so I guess this short comment is in line with how much/little doc we want in the example manifests.

Reply:
Yeah. @FrankOrigo intends to reduce the amount of comments in the cluster/application-.yaml files. I agree. I'ma add the improvements he wants to the cluster and application declarations asap. They include a link to the userdocs